src/data/GraphOnChipContext.tsx

```diff
@@ -150,23 +131,7 @@ const GraphOnChipProvider: React.FC<{ children: ReactNode }> = ({ children }) =>
         );
     }, [state.graphs]);
 
-    const getActiveGraphRelationship = useCallback(() => {
-        return state.graphs.get(state.visitedGraphsHistory[state.currentGraphIndex] ?? '');
-    }, [state]);
-
-    const getActiveGraphOnChip = useCallback(() => {
-        return state.graphOnChipList[state.visitedGraphsHistory[state.currentGraphIndex] ?? ''];
-    }, [state]);
-
-    const getActiveGraphOnChipListForTemporalEpoch = useCallback(() => {
-        const epoch = getActiveGraphRelationship()?.temporalEpoch;
-        const listOfGraphRel: GraphRelationship[] = [...state.graphs.values()].filter((graph: GraphRelationship) => {
-            return graph.temporalEpoch === epoch;
-        });
-        return listOfGraphRel.map((graph) => {
-            return { graph, graphOnChip: state.graphOnChipList[graph.name] };
-        });
-    }, [getActiveGraphRelationship, state.graphOnChipList, state.graphs]);
+    const getActiveGraphOnChip = useCallback(() => undefined, []);
```

Comment:
you cant deprecate and break a method at the same time. deprecated should still work.
instead remove it all together if its returning undefined